Sources/NIO/EventLoop.swift

```diff
@@ -492,7 +499,10 @@ extension EventLoop {
     ///     - task: The closure that will be executed.
     /// - return: `RepeatedTask`
     @discardableResult
-    public func scheduleRepeatedTask(initialDelay: TimeAmount, delay: TimeAmount, notifying promise: EventLoopPromise<Void>? = nil, _ task: @escaping (RepeatedTask) -> EventLoopFuture<Void>) -> RepeatedTask {
+    public func scheduleRepeatedAsyncTask(initialDelay: TimeAmount,
```

Comment:
do we also need to update the tests so it not use the old deprecated name ?

Reply:
@normanmaurer wow, good catch. There was no test (contributor contribution) for that method. But all synchronous ones use the async one to execute. Added a good test.